cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,84 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+  url = '/settings';
+
+  editUsername (username) {
+    cy.getByDataCy('usernameSettings')
+      .type('{selectAll}' + `${username}`);
+  };
+
+  editBio (bio) {
+    cy.getByDataCy('bioSettings')
+      .type('{selectAll}' + `${bio}`);
+  };
+
+  editEmail (email) {
+    cy.getByDataCy('emailSettings')
+      .type('{selectAll}' + `${email}`);
+  };
+
+  editPassword (password) {
+    cy.getByDataCy('passwordSettings')
+      .type('{selectAll}' + `${password}`);
+  };
+
+  submitSettings () {
+    cy.getByDataCy('submitSettings')
+      .click();
+  };
+
+  assertUsername(username) {
+    cy.getByDataCy(`/profile/${username}`)
+      .should('contain', username);
+  };
+
+  assertBio(bio) {
+    cy.getByDataCy('/settings')
+      .click();
+    cy.getByDataCy('bioSettings')
+      .should('contain', bio);
+  };
+
+  logOut() {
+    cy.getByDataCy('/settings')
+      .click();
+    cy.get('.btn-outline-danger')
+      .click();
+  };
+
+  assertLogOut() {
+    cy.getByDataCy('/user/login')
```

Comment:
it doesn't look like data-cy attr.
fix all cases where u use getbydatacy but did not use data attr